nucypher/network/middleware.py

```diff
@@ -203,15 +203,8 @@ def get_certificate(self, host, port, timeout=3, retry_attempts: int = 3, retry_
                                                          backend=default_backend())
             return certificate
 
-    def propose_arrangement(self, node, arrangement):
-        response = self.client.post(node_or_sprout=node,
-                                    path="consider_arrangement",
-                                    data=bytes(arrangement),
-                                    timeout=120)  # TODO: What is an appropriate timeout here?
-        return response
-
-    def revoke_arrangement(self, ursula, revocation):
-        # TODO: Implement revocation confirmations
+    def revoke_node(self, ursula, revocation):
```

Comment:
Not quite happy with this name. Is it rather `revoke_from_node` or something?

Reply:
I think `revoke_from_node` or `revoke_on_node` (or `_at_node`? not sure) would be closer to what's happening. Or, perhaps, `request_revocation`?

Reply:
Out of all of those, I like `request_revocation` the best, but it's a bit redundant considering this is a network request.  What's misleading about `revoke_node`?